.openzeppelin/rinkeby.json

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
What are these files generated by? Do we need them?

Reply:
This was just testing.
I have deleted it.